packages/common-types/src/baseTypes/aggregateTypes.ts

```diff
@@ -14,6 +14,7 @@ import {
   ShareStoreMap,
   ShareStorePolyIDShareIndexMap,
 } from "../base";
+import { KeyType } from "../utils";
```

Comment:
should we name it curve instead of keytype

Reply:
done

Reply:
Sorry, Torus js is using KeyType.
https://github.com/torusresearch/torus.js/pull/126/files
let keep consistency